### PR TITLE
configuration: added showing session data in Tracy bar

### PR DESCRIPTION
**Pull request by petrparolek**
added miissing configuration

### CODE REVIEW

File: cs/configuring.texy [hunk 1]
```diff
@@ -297,6 +297,13 @@ Pokud chceme platnost session (nebo autentizace) rozšířit na subdomémy, nast
 		cookieDomain: 'example.com'
 \--
 
+Pokud chceme zobrazit obsah sessiony v panelu:
```
Comment (f3l1x) [inline cs/configuring.texy]: `Pokud chceme zobrazit obsah session v Debugger baru:`

By bylo asi lepsi. :-)

File: cs/configuring.texy [hunk 2]
```diff
@@ -297,6 +297,13 @@ Pokud chceme platnost session (nebo autentizace) rozšířit na subdomémy, nast
...
+
+/--neon
+	session:
+		debugger: TRUE
```
Comment (dg) [inline cs/configuring.texy]: it should be `true`

**Comment by petrparolek** [comment]
vylepšeno

**Comment by petrparolek** [comment]
ping @f3l1x

File: en/configuring.texy [hunk 3]
```diff
@@ -293,6 +293,13 @@ If you want to extend the validity of a session (or authentication) for subdomai
 		cookieDomain: 'example.com'
 \--
 
+If you want to show session data in bar:
+
+/--neon
+	session:
+		debugger: TRUE
```
[the remark this answers is not in the document]
Comment (petrparolek) [reply en/configuring.texy]: fixed

**Comment by petrparolek** [comment]
fixed

File: en/configuring.texy [hunk 4]
```diff
@@ -293,6 +293,13 @@ If you want to extend the validity of a session (or authentication) for subdomai
 		cookieDomain: 'example.com'
 \--
 
+If you want to show session data in bar:
```
Comment (f3l1x) [inline en/configuring.texy]: `...data in Debug bar`.

Comment (petrparolek) [reply en/configuring.texy]: fixed, thx

**Comment by petrparolek** [comment]
fixed

**Comment by dg** [comment]
Great, thanks

**Comment by f3l1x** [comment]
Cool.

**Pull request merged**